docs/modules/ROOT/pages/features/index.adoc

```diff
@@ -1,6 +0,0 @@
-= Features
-
-xref:features/syntax-highlighting.adoc[Syntax Highlighting]::
-Have your code highlighted as you type.
-xref:features/bracket-matching.adoc[Bracket Matching]::
-Let the editor take care of indentation and closing brackets.
```

Comment:
Since the list of features grows over time, it's somewhat extra to have full pages here to describe every feature. It would also be an unnecessary maintenance burden to try to keep these docs up to date.